ui/css/treeherder-resultsets.css

```diff
@@ -73,11 +73,7 @@
   background-color: transparent;
   padding-left: 9px;
   padding-right: 10px;
-}
```

Comment:
This file could be renamed to `treeherder-pushes` at some point, since `resultsets` is deprecated nomenclature.  Coincidentally, "Deprecated Nomenclature" was also my favorite band in High School.  No, you won't find them.  Way indie...